org.codecoop.kjo.bookletimposer.yaml

```diff
@@ -0,0 +1,84 @@
+id: org.codecoop.kjo.bookletimposer
+version: 1.4.1
+branch: stable
+runtime: org.gnome.Platform
+runtime-version: '3.32'
+sdk: org.gnome.Sdk
+command: bookletimposerwrapper.sh
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --filesystem=home
+modules:
+  - shared-modules/python2.7/python-2.7.json
+
+  - name: pyPdf2
```

Comment:
Have you generated these dependencies with [pip generator](https://github.com/flatpak/flatpak-builder-tools/tree/master/pip)?

Reply:
No i try to use it but in my system it failed so i did it from scratch